internal/acctest/virtualhostnames/virtual_host_names_test.go

```diff
@@ -55,6 +55,9 @@ func testAccVirtualHostNames(resourceName string, resourceModel virtualHostNames
 	return fmt.Sprintf(`
 resource "pingfederate_virtual_host_names" "%[1]s" {
   virtual_host_names = %[2]s
+}
+data "pingfederate_virtual_host_names" "%[1]s" {
+  virtual_host_names = pingfederate_virtual_host_names.%[1]s.virtual_host_names
```

Comment:
You won't need to do this after making virtual_host_names computed

Reply:
Sorry, I mean just removing line 60 here. We still want the data source in the acceptance test.